components/authz-service/storage/postgres/migration/migration.go

```diff
@@ -106,7 +109,9 @@ func (c *Config) Migrate(dataMigConf datamigration.Config) error {
 		if err != nil {
 			return errors.Wrapf(err, "failed to set IAM v2 migration_status to %s", constants_v2.EnumInProgress)
 		}
+		l.Info("Migrating from IAM v1 to v2...")
```

Comment:
Is it weird to draw all this attention to v1 -> v2? For new users, there has never been a v1. I considered renaming this function and it fell through the cracks, but we're only "migrating v1 -> v2" within the `MigrateToV2` function _if_ it's not a fresh install. Otherwise, we're really, er, "Creating base IAM roles and policies" . . . ? If you'd like to do a rename and push this logging into that function, I suppose that is fine. Otherwise, I'd modify this message to be something more like "Setting up IAM basics" or something.

Reply:
But his will only get run if someone is on v1, right? Or are you saying that out-of-the-box a fresh install will still start in v1, and will run this bit of code?

Reply:
Yes, the system understands "isOnV1" as "hasNotUpgradedToV2", including fresh installs.

Reply:
You could use the bool `migrateV1Policies` as an indicator for when we'll be doing *actual* v1-v2 migration work (the v1 policies)